docs/advanced/subscriptions.md

```diff
@@ -40,74 +40,64 @@ When we define this function it must return an "Observable-like" object, which n
 [Observable spec](https://github.com/tc39/proposal-observable), which comes down to having an
 object with a `.subscribe()` method accepting an observer.
 
-## Setting up `subscriptions-transport-ws`
+## Setting up `graphql-ws`
 
-If your GraphQL API is using [the Apollo Server](https://www.apollographql.com/docs/apollo-server/),
-you'll be able to use [Apollo's `subscriptions-transport-ws`
-package](https://github.com/apollographql/subscriptions-transport-ws).
+For backends supporting `graphql-ws`, we recommend using the [graphql-ws](https://github.com/enisdenjo/graphql-ws) client.
 
 ```js
-import { Client, defaultExchanges, subscriptionExchange } from 'urql';
-import { SubscriptionClient } from 'subscriptions-transport-ws';
+import { createClient, defaultExchanges, subscriptionExchange } from 'urql';
+import { createClient as createWSClient } from 'graphql-ws';
 
-const subscriptionClient = new SubscriptionClient('ws://localhost/graphql', { reconnect: true });
+const wsClient = createWSClient({
+  url: 'ws://localhost/graphql',
+});
 
-const client = new Client({
+const client = createClient({
   url: '/graphql',
   exchanges: [
     ...defaultExchanges,
     subscriptionExchange({
-      forwardSubscription(operation) {
-        return subscriptionClient.request(operation);
-      },
+      forwardSubscription: (operation) => ({
+        subscribe: (sink) => ({
+          unsubscribe: wsClient.subscribe(operation, sink),
+        }),
+      }),
     }),
   ],
 });
 ```
 
-In this example, we're creating a `SubscriptionClient`, are passing in a URL and some parameters,
-and are using the `SubscriptionClient`'s `request` method to create a Subscription Observable, which
-we return to the `subscriptionExchange` inside `forwardSubscription`.
+[Read more on the `graphql-ws` README.](https://github.com/enisdenjo/graphql-ws/blob/master/README.md)
 
-[Read more about `subscription-transport-ws` on its README.](https://github.com/apollographql/subscriptions-transport-ws/blob/master/README.md)
+## Setting up `subscriptions-transport-ws`
 
-## Setting up `graphql-ws`
+For backends supporting `subscriptions-transport-ws`, [Apollo's `subscriptions-transport-ws`
+package](https://github.com/apollographql/subscriptions-transport-ws) can be used.
 
-If your GraphQL API server is using [graphql-ws](https://github.com/enisdenjo/graphql-ws),
-you'll be able to use it here too!
+> Note: `subscriptions-transport-ws` is deprecated, consider using `graphql-ws` (above)
```

Comment:
```suggestion
> The `subscriptions-transport-ws` package isn't actively maintained. If your API supports the new protocol or you can swap the package out, consider using [`graphql-ws`](#setting-up-graphql-ws) instead.
```

It never has been put into official deprecation or archival, so I'd love to keep this generic, especially since the GraphQL over HTTP spec extension moving to standardise the protocol anyway

Reply:
Can do that if you want - I was basing that deprecation status on what is in the [README](https://github.com/apollographql/subscriptions-transport-ws)

> The subscriptions-transport-ws library is not being actively maintained. It is recommended that you use the graphql-ws library instead.